mathml/presentation-markup/operators/mo-form-dynamic.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>&lt;mo&gt; dynamic form</title>
+    <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#tokenmo">
+    <meta name="assert" content="Verifies dynamic behavior of form.">
+    <link rel="match" href="mo-form-dynamic-ref.html">
+
+    <script>
+      function test()
+      {
+        for (var i = 1; i <= 6; i++) {
+          var row = document.getElementById("row" + i);
+          var x = document.getElementById("x" + i);
+          x.parentNode.removeChild(x);
+          row.insertBefore(x, row.firstElementChild);
```

Comment:
Can you make this a reftest-wait test?
(see https://github.com/web-platform-tests/wpt/pull/16575)

Reply:
Done.